starport/cmd/account.go

```diff
@@ -39,26 +39,15 @@ Starport uses accounts to interact with the Starport Network blockchain, use an
 	return c
 }
 
-func printAccounts(cmd *cobra.Command, accounts ...cosmosaccount.Account) {
-	w := &tabwriter.Writer{}
-	w.Init(os.Stdout, 0, 8, 0, '\t', 0)
-
-	if len(accounts) == 0 {
-		return
-	}
-
-	fmt.Fprintln(w, "name\taddress\tpublic key")
-
+func printAccounts(cmd *cobra.Command, accounts ...cosmosaccount.Account) error {
+	var accEntries [][]string
 	for _, acc := range accounts {
-		fmt.Fprintf(w, "%s\t%s\t%s\n",
-			acc.Name,
-			acc.Address(getAddressPrefix(cmd)),
-			acc.PubKey(),
-		)
+		accEntries = append(accEntries, []string{acc.Name, acc.Address(getAddressPrefix(cmd)), acc.PubKey()})
 	}
-
-	fmt.Fprintln(w)
-	w.Flush()
+	if err := entrywriter.Write(os.Stdout, []string{"name", "address", "public key"}, accEntries...); err != nil {
```

Comment:
Maybe we should keep one pattern for all headers

```suggestion
	if err := entrywriter.Write(os.Stdout, []string{"Name", "Address", "Public Key"}, accEntries...); err != nil {
```